caiman/motion_correction.py

```diff
@@ -69,15 +69,7 @@
 except:
     def profile(a): return a
 
-opencv = True
-
-try:
-    import pycuda.gpuarray as gpuarray
-    import pycuda.driver as cudadrv
-    import atexit
-    HAS_CUDA = True
-except ImportError:
-    HAS_CUDA = False
+opencv = True # FIXME How is the user meant to be able to interact with this?!
```

Comment:
Yea this is weird. Also skimage uses opencv under the hood for many things so I wonder what this actually does.

Reply:
This may be a remnant of the spyder era, when the line between library code and code the user might edit was very thin. It was fairly unstructured and messy.

Maybe before this is ready for commit I'll stare at this a lot more and maybe dig through the commit history - it may be better to clean this up than just add a note. Although maybe that's better done as a separate diff.

Reply:
The code is ancient, introduced in 2e7f1a2719f38cd2016038e27f398fa7d2f9a339 (December 2016) during the spyder era when the code was not yet really following library semantics.

We probably should remove the (now probably long-untested) opencv=False code path.